tests/test_wasapi_client.py

```diff
@@ -504,7 +540,30 @@ def test_run_WASAPIDownloadError(self, mock_download):
         assert result_q.qsize() == 2
         assert log_q.qsize() == 2
         for _ in (1, 2):
-            assert result_q.get() == ('failure', self.FILE_DATA['filename'])
+            assert result_q.get() == ('failure', self.filename)
+
+    def test_run_file_already_verified(self):
```

Comment:
I think I'm missing something, I don't quite get what this test is actually testing.

Reply:
What I wanted it to test is that `verify_file` was not called, since the returned object from `download_file` indicates the  file was theoretically already verified in `download_file`

Reply:
What do you think about maybe changing the docstring? Perhaps something along the lines of "Test a downloaded file is not verified twice." or something along those lines?

Reply:
done. 48c41fb